subsys/net/lib/openthread/platform/radio.c

```diff
@@ -1223,3 +1252,23 @@ otError otPlatRadioConfigureEnhAckProbing(otInstance *aInstance, otLinkMetrics a
 }
 
 #endif /* CONFIG_OPENTHREAD_LINK_METRICS_SUBJECT */
+
+otError otPlatRadioSetChannelMaxTransmitPower(otInstance *aInstance, uint8_t aChannel,
+					      int8_t aMaxPower)
+{
+	ARG_UNUSED(aInstance);
+	otError error = OT_ERROR_NONE;
```

Comment:
The variable does not seem to be used at all, we could return `OT_ERROR_NONE` directly at the end.

Reply:
fixed